### DISCUSSION TITLE
Convert model to active model

### DISCUSSION THREAD

**Discussion by zys864** [SeaQL/sea-orm#930] [Q&A] [answered]
## Motivation
Activemodel don't support Serialize, Deserialize. I deserialize json to Model and wan to insert to db.
There is any convinient wany to convert model to Active model?

**Comment by billy1624** [comment, marked as answer]
Hey @zys864, you could easily convert any `Model` into `ActiveModel` with `into_active_model` method.
_https://docs.rs/sea-orm/0.9.1/sea_orm/entity/trait.IntoActiveModel.html#tymethod.into_active_model_

**Comment by zys864** [comment]
thanks,it works.

**Comment by billy1624** [comment]
Welcome!

**Comment by GaspardCulis** [comment]
Is it possible the other way around ? ie. get `Model` from `ActiveModel` ?

**Reply by Huliiiiii** [reply]
What is your use case? 
What do you expect to get from an unset field?

**Reply by GaspardCulis** [reply]
Maybe a `Default` value... Ok I guess that ain't possible for all cases and should be done manually
